src/modules/chat_commands/index.js

```diff
@@ -0,0 +1,247 @@
+        case 'localsuboff':
+            const subsOnly = !command.endsWith('off');
+            chat.subsOnly(subsOnly);
+            twitch.sendChatAdminMessage(`Local subs-only mode ${subsOnly ? 'enabled' : 'disabled'}.`);
+            break;
+
+        // fun
+        case 'shrug':
+            return `${messageParts.join(' ')} ¯\\_(ツ)_/¯`;
+        case 'squishy':
+            return 'notsquishY WHEN YOU NEED HIM notsquishY IN A JIFFY notsquishY USE THIS EMOTE notsquishY TO SUMMON SQUISHY notsquishY';
+
+        // misc
+        case 'join':
+        case 'part':
+            command === 'join' ? anonChat.join() : anonChat.part();
+            break;
+
+        // info
+        case 'chatters':
+            tmiAPI.getChatterCount(channel.name)
+                .then(chatterCount => twitch.sendChatAdminMessage(`Current Chatters: ${chatterCount.toLocaleString()}`));
+            break;
+        case 'followed':
+            const currentUser = twitch.getCurrentUser();
+            if (!currentUser) break;
+            twitchAPI.get(`users/${currentUser.id}/follows/channels/${channel.id}`)
+                .then(({created_at}) => {
+                    const since = moment(created_at);
+                    twitch.sendChatAdminMessage(`You followed ${channel.displayName} ${since.fromNow()} (${since.format('LLL')})`);
+                })
+                .catch(() => twitch.sendChatAdminMessage(`You do not follow ${channel.displayName}.`));
+            break;
+        case 'follows':
+            twitchAPI.get(`channels/${channel.id}`)
+                .then(({followers}) => twitch.sendChatAdminMessage(`Current Followers: ${followers.toLocaleString()}`))
+                .catch(() => twitch.sendChatAdminMessage('Could not fetch follower count.'));
+            break;
+        case 'viewers':
+            twitchAPI.get(`streams/${channel.id}`)
+                .then(({stream}) => {
+                    const viewers = stream ? stream.viewers : 0;
+                    twitch.sendChatAdminMessage(`Current Viewers: ${viewers.toLocaleString()}`);
+                })
+                .catch(() => twitch.sendChatAdminMessage('Could not fetch stream.'));
+            break;
+        case 'uptime':
+            twitchAPI.get(`streams/${channel.id}`)
+                .then(({stream}) => {
+                    const startedTime = stream ? new Date(stream.created_at) : null;
+                    if (!startedTime) {
+                        twitch.sendChatAdminMessage('Stream is not live');
+                        return;
+                    }
+
+                    const secondsSince = Math.round((Date.now() - startedTime.getTime()) / 1000);
+                    twitch.sendChatAdminMessage(`Current Uptime: ${secondsToLength(secondsSince)}`);
+                })
+                .catch(() => twitch.sendChatAdminMessage('Could not fetch stream.'));
+            break;
+
+        // misc
+        case 'localunpin':
+            chat.dismissPinnedCheer();
```

Comment:
we should just remove this command since it's duplicate functionality of the dismiss button on them directly